content/blog/hackaton-2022-building-a-pong.md

```diff
@@ -0,0 +1,212 @@
+
+---
+Title: Hackapong fun at the shop!
+Date: 2022-11-04 13::55:00
+Author: Jean-Philippe Lamebrt - alex Hoguin
+Category: WDA
+Tags: hackathon, wda
+Slug: hackaton-2022-building-a-pong
+Status: published
+---
```

Comment:
Also, filename should be `hackathon-2022-building-a-pong.md` (missing H in hachkathon)

```suggestion
---
Title: Hackapong Fun at the Shop!
Date: 2022-11-04 13:55:00
Author: Jean-Philippe Lambert & Alex Hoguin
Category: Hackathon
Tags: hackathon, dtmf, websocket
Slug: hackathon-2022-building-a-pong
Status: published
---
```

Reply:
Many things in this suggestion is not fixed yet.

Reply:
done